packages/nhost-js/src/utils/types.ts

```diff
@@ -0,0 +1,57 @@
+import type { AxiosResponse } from 'axios'
```

Comment:
I would rather keep types in `src/types` as it is the most common place to put typings that I've seen in ts libraries so far